components/automate-ui/src/app/modules/desktop/daily-check-in/daily-check-in.component.html

```diff
@@ -3,7 +3,6 @@
     <main>
       <div class="page-title">Daily Check-in</div>
       <div class="update-time">Updated {{ lastUpdatedMessage }}</div>
-
```

Comment:
[nit] - revert this file....or don't.  It's getting changed out anyways.  But wanted to make sure I pointed it out in case you felt the need to.